InnerEye/ML/configs/classification/CovidModel.py

```diff
@@ -237,45 +239,118 @@ def compute_and_log_metrics(self,
-                predictions.append(target_df[LoggingColumns.ModelOutput.value])
-                labels.append(target_df[LoggingColumns.Label.value])
+                predictions.append(target_df[output_prefix].item())
+                labels.append(target_df[label_prefix].item())
 
-            return pd.DataFrame.from_dict({LoggingColumns.Patient.value: [df.iloc[0][LoggingColumns.Patient.value]],
-                                           LoggingColumns.ModelOutput.value: [np.argmax(predictions)],
-                                           LoggingColumns.Label.value: [np.argmax(labels)]})
+            pred_cvx03vs12 = predictions[1] + predictions[2]
+            label_cvx03vs12 = 1 if labels[1] or labels[2] else 0
 
-        def get_accuracy(df: pd.DataFrame) -> float:
-            df = df.groupby(LoggingColumns.Patient.value, as_index=False).apply(get_labels_and_predictions).reset_index(
-                drop=True)
-            return (df[LoggingColumns.ModelOutput.value] == df[LoggingColumns.Label.value]).mean()  # type: ignore
+            if (predictions[0] + predictions[3]) != 0:
+                pred_cvx0vs3 = predictions[3] / (predictions[0] + predictions[3])
+            else:
+                pred_cvx0vs3 = np.NaN
+            label_cvx0vs3 = 0 if labels[0] else 1 if labels[3] else np.NaN
 
-        train_metrics = get_output_csv_path(ModelExecutionMode.TRAIN)
-        val_metrics = get_output_csv_path(ModelExecutionMode.VAL)
-        test_metrics = get_output_csv_path(ModelExecutionMode.TEST)
+            if (predictions[1] + predictions[2]) != 0:
+                pred_cvx1vs2 = predictions[2] / (predictions[1] + predictions[2])
+            else:
+                pred_cvx1vs2 = np.NaN
+            label_cvx1vs2 = 0 if labels[1] else 1 if labels[2] else np.NaN
 
-        msg = f"Multiclass Accuracy Train: {get_accuracy(pd.read_csv(train_metrics))}\n" if train_metrics.exists() else ""
-        msg += f"Multiclass Accuracy Val: {get_accuracy(pd.read_csv(val_metrics))}\n" if val_metrics.exists() else ""
-        msg += f"Multiclass Accuracy Test: {get_accuracy(pd.read_csv(test_metrics))}\n" if test_metrics.exists() else ""
+            return pd.DataFrame.from_dict({LoggingColumns.Patient.value: [df.iloc[0][LoggingColumns.Patient.value]],
+                                           output_prefix: [np.argmax(predictions)],
+                                           label_prefix: [np.argmax(labels)],
+                                           f"{output_prefix}_CVX03vs12": pred_cvx03vs12,
+                                           f"{label_prefix}_CVX03vs12": label_cvx03vs12,
+                                           f"{output_prefix}_CVX0vs3": pred_cvx0vs3,
+                                           f"{label_prefix}_CVX0vs3": label_cvx0vs3,
+                                           f"{output_prefix}_CVX1vs2": pred_cvx1vs2,
+                                           f"{label_prefix}_CVX1vs2": label_cvx1vs2})
```

Comment:
minor suggestion: as these keys/column names are used multiple times, would it be helpful to define them as variables at the top, e.g. `output_03vs12_col` etc.?